tests/Unit/API/Authentication/PushedAuthorizationRequestTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types=1);
+
+use Auth0\SDK\API\Authentication;
+use Auth0\SDK\API\Management\Jobs;
+use Auth0\SDK\Auth0;
+use Auth0\SDK\Configuration\SdkConfiguration;
+use Auth0\SDK\Exception\Authentication\ParResponseException;
+use Auth0\SDK\Exception\NetworkException;
+use Auth0\Tests\Utilities\HttpResponseGenerator;
+use Auth0\Tests\Utilities\MockDomain;
+use Auth0\Tests\Utilities\TokenGenerator;
+
+uses()->group('authentication', 'authentication.pushedAuthorizationRequest');
+
+beforeEach(function(): void {
+    $this->secret = uniqid();;
```

Comment:
```suggestion
    $this->secret = uniqid();
```
There's a double semicolon here.

Reply:
Fixed! Thanks